modules/mssql/mssql.go

```diff
@@ -41,6 +43,45 @@ func WithPassword(password string) testcontainers.CustomizeRequestOption {
 	}
 }
 
+// WithScripts adds SQL scripts to be executed after the container is ready.
+// The scripts are executed in the order they are provided using sqlcmd tool.
+func WithScripts(scripts ...string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) error {
+		hooks := make([]testcontainers.ContainerHook, 0, len(scripts))
+		for _, script := range scripts {
+			hook := func(ctx context.Context, c testcontainers.Container) error {
+				password := defaultPassword
+				if req.Env["MSSQL_SA_PASSWORD"] != "" {
+					password = req.Env["MSSQL_SA_PASSWORD"]
+				}
+
+				targetPath := "/tmp/" + filepath.Base(script)
+				if err := c.CopyFileToContainer(ctx, script, targetPath, 0o644); err != nil {
+					return err
+				}
+
+				cmd := testcontainers.NewRawCommand([]string{
+					GetSQLCmdPath(req.Image),
+					"-S", "localhost",
+					"-U", defaultUsername,
+					"-P", password,
+					"-No",
+					"-i", targetPath,
+				})
+				_, _, err := c.Exec(ctx, cmd.AsCommand(), cmd.Options()...)
+				return err
```

Comment:
suggestion: wrap the error so its clear about the cause
```suggestion
				if _, _, err := c.Exec(ctx, cmd.AsCommand(), cmd.Options()...); err != nil {
					return fmt.Errorf("exec script %q: %w", script, err)
				}
				return nil
```